action/protocol/poll/protocol.go

```diff
@@ -124,7 +124,7 @@ func NewProtocol(
 	getkickoutList GetKickoutList,
 	getUnproductiveDelegate GetUnproductiveDelegate,
 	electionCommittee committee.Committee,
-	stakingV2 *staking.Protocol,
+	stakingProto *staking.Protocol,
```

Comment:
nit: rename to nativeStk?
proto might lead to think protobuf

Reply:
i think it is fine. staking.Protocol -> stakingProto

Reply:
nativeStk may confused with poll.nativestaking and poll.nativestakingv2 as well.